projects/plugins/jetpack/_inc/lib/core-api/wpcom-endpoints/class-wpcom-rest-api-v2-endpoint-admin-menu.php

```diff
@@ -75,6 +75,8 @@ public function get_item_permissions_check( $request ) { // phpcs:ignore Generic
 	 * @return WP_REST_Response|WP_Error Response object on success, or WP_Error object on failure.
 	 */
 	public function get_item( $request ) { // phpcs:ignore Generic.CodeAnalysis.UnusedFunctionParameter, VariableAnalysis.CodeAnalysis.VariableAnalysis.UnusedVariable
+		require_once JETPACK__PLUGIN_DIR . 'modules/masterbar/admin-menu/load.php';
```

Comment:
This doesn't work for me on the wpcom side. 
```
require_once(/home/wpcom/public_html/wp-content/mu-plugins/jetpack/modules/masterbar/admin-menu/load.php): failed to open stream: No such file or directory
```
`require_once ABSPATH . 'wp-content/mu-plugins/masterbar/admin-menu/load.php';` seems working from wpcom.

Reply:
Oh right, `JETPACK__PLUGIN_DIR` is likely undefined there. Let me see if there is any shared util that allows to load a file in both envs.

Reply:
2675046